CRM/Core/Error.php

```diff
@@ -600,7 +600,15 @@ public static function debug_log_message($message, $out = FALSE, $comp = '', $pr
     }
     $file_log->close();
 
-    if (!empty($config->userFrameworkLogging)) {
+    if (!isset(\Civi::$statics[__CLASS__]) || !isset(\Civi::$statics[__CLASS__]['userFrameworkLogging'])) {
```

Comment:
(a) In my experience/environment, the first `isset()` hasn't been necessary. If it is necessary, then we probably need to cleanup a lot of others. (Also: curious to know what environment you have where it's necessary.)

(b) If it is necessary to explicitly check both, then you can probably combine the two `isset()`s:

```php
if (!isset(\Civi::$statics[__CLASS__], \Civi::$statics[__CLASS__]['userFrameworkLogging']))
```